source/system/system.go

```diff
@@ -153,6 +170,16 @@ func splitVersion(version string) map[string]string {
 	return components
 }
 
+// Read /sys/devices/virtual/dmi/id attribute
+func getDmiIdAttribute(name string) (string, error) {
```

Comment:
Usually is a good practice to use ID
`getDmiIdAttribute` --> `getDmiIDAttribute`
But is not a blocker for this patch

Reply:
@ArangoGutierrez
Thanks! Updated both